src/form/input/Input.tsx

```diff
@@ -8,6 +8,7 @@ export type InputProps = Omit<
   "disabled" | "name"
```

Comment:
Let's omit `className` too so that users of the component are forced to use `customClassName` instead
```suggestion
  "disabled" | "name" | "className"
```